packages/circuits/test/verifyEpochKey.test.ts

```diff
@@ -81,29 +80,30 @@ describe('Verify Epoch Key circuits', function () {
 
             await executeCircuit(circuit, circuitInputs)
             const startTime = new Date().getTime()
-            const { proof, publicSignals } = await genProofAndPublicSignals(
-                Circuit.verifyEpochKey,
-                circuitInputs
-            )
+            const { proof, publicSignals } =
```

Comment:
Yea that's odd. Here we have this second assertion:

```
const formatProof = formatProofForVerifierContract(proof)
const snarkjsProof = formatProofForSnarkjsVerification(formatProof)
isValid = await defaultProver.verifyProof(
  Circuit.verifyEpochKey,
  publicSignals,
  snarkjsProof
)
```

We can't do this if we use the utils function because it doesn't return `proof`/`publicSIgnals`

Reply:
here is a test of two `formatProof`s
Or we can use another unit to test it 😅